.devcontainer/devcontainer.json

```
@@ -1,21 +1,27 @@
 {
   "hostRequirements": {
```

nit: can you add `$schema` here to make for easier editing

```suggestion
  "$schema": "https://raw.githubusercontent.com/devcontainers/spec/main/schemas/devContainer.schema.json",
  "hostRequirements": {
```

This comes from: https://containers.dev/implementors/json_schema/